src/controllers/taxonomias-controller.js

```diff
@@ -286,9 +286,8 @@ export const cadastrarSubfamilia = (request, response, next) => {
 
             return familiaEncontrada;
         })
-        .then(familia => Subfamilia.create({ nome,
+        .then(Subfamilia.create({ nome,
```

Comment:
@MatCoitinho aqui você ainda precisa manter a função, mesmo que você não faça uso do parâmetro, no caso


```suggestion
        .then(() => Subfamilia.create({ nome,
```